### PR TITLE
docs: ✏️ Rework of Docs to Address Blueprints & Remove Feedback

### PR DESCRIPTION
**Pull request by richard-wpengine**
Internal form links, Gmail limitation, and parenthesis in headers all also dropped

✅ Closes: ORN-1012

### CODE REVIEW

**Review by ToughCrab24 — APPROVED** [review]

**Comment by ToughCrab24** [comment]
@thed00de are you adding a reference to `atlas-search@wpengine.com` the introduction.mdx?

**Comment by thed00de** [comment]
> @thed00de are you adding a reference to `atlas-search@wpengine.com` the introduction.mdx?

no
LGTM now

File: atlas-search/introduction.mdx [hunk 1]
```diff
@@ -1,93 +1,102 @@
...
-        - Limited to 1 instance per Gmail account.
-    + Automatic-enrollment as part of Atlas “eval” account.
-2. You have received an Atlas Search setup email with the following:
-    + A link to this documentation
-    + A link to download the latest [Atlas Search](https://wordpress.org/plugins/atlas-search/) plugin, from WP Plugin Store.
-    + Your own personal Atlas-site URL and Access Token.
+### Installing the Atlas Search Plugin
+The Atlas Search plugin can be installed manually but will more likely be installed implicitly as part of our Atlas Blueprints setup.
   
+#### Installation via Blueprints
```
Comment (JEverhart383) [inline atlas-search/introduction.mdx:50]: I think it's good to break out the blueprint vs. non-blueprint users here for instructions

**Review by JEverhart383 — APPROVED** [review]
There are some good clarifications and revisions in here. Approving and pushing to staging so we can review there.

**Pull request merged**